package/gluon-mesh-vpn-core/src/respondd.c

```diff
@@ -42,7 +42,7 @@ static struct json_object * get_bandwidth_limit(void) {
 
 	struct uci_context *ctx = uci_alloc_context();
 	if (!ctx)
-		goto disabled_nofree;
+		goto disabled;
```

Comment:
This was previously more efficient. Now you have two ifs...
@blocktrron Why should he have changed this? What's the problem with having two labels?
Do you mean it is more efficient, because of branch prediction?

Reply:
In it's current state it doesn't really matter. However, in case this code is extended and more memory is allocated on the heap, freeing everything on completion is more readable instead of having multiple locations to jump to.

Performance-wise, these additional checks don't matter that much, given requests for responss from the running node are multiplexed anyhow.